lib-opensource/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -550,10 +550,19 @@ listeners(["stop", _Proto, ListenOn]) ->
     end,
     stop_listener(emqx_listeners:find_by_listen_on(ListenOn1), ListenOn1);
 
+listeners(["restart", Identifier]) ->
+    case emqx_listeners:restart_listener(Identifier) of
+        ok ->
+            emqx_ctl:print("Restarted ~s listener successfully.~n", [Identifier]);
+        {error, Error} ->
+            emqx_ctl:print("Failed to restart ~s listener error:~p~n", [Identifier, Error])
```

Comment:
```suggestion
            emqx_ctl:print("Failed to restart ~s listener error: ~0p~n", [Identifier, Error])
```

Reply:
Yea, I'm gonna adjust it to `Failed to restart ~s listener: ~0p~n` like other error messages